### DISCUSSION TITLE
Connections v7 or v8 ?

### DISCUSSION THREAD

**Discussion by MagnaFrisia90** [HCL-TECH-SOFTWARE/connections-automation#246] [Q&A] [answered]
The v7 documentation references this git project - eventhough the last change for this project indicates the ansible script is for v8 envoirnments.
Can I use the ansible playbook for Component Pack 7.0.0.2 & Connections v7 ?

**Comment by umeli** [comment, marked as answer]
The playbooks work for V7 too. Just have a look at the examples https://github.com/HCL-TECH-SOFTWARE/connections-automation/tree/main/environments/examples/cnx7
